src/com/palmergames/bukkit/towny/TownyAPI.java

```diff
@@ -482,6 +482,13 @@ public TownBlockStatus hasNationZone(WorldCoord worldCoord) {
 			
 			nearestTown = nearestTownblock.getTown();
 			
+			//If nearest town has an in-progress siege, and war disables config is true, nationzone is disabled.
+			if(TownySettings.getWarSiegeEnabled()
+				&& TownySettings.getNationZonesWarDisables()
+				&& nearestTown.hasSiege()
+				&& nearestTown.getSiege().getStatus().isActive())	{
+				return TownBlockStatus.UNCLAIMED_ZONE;
+			}
```

Comment:
This needs to be extracted from `TownBlockStatus#hasNationZone(WorldCoord)` or replaced with something that war plugins can tie into.